aws/ec2.go

```diff
@@ -768,3 +775,33 @@ func GetEC2ResourceNameTagValue(tags []*ec2.Tag) (string, error) {
 	}
 	return "", fmt.Errorf("Resource does not have Name tag")
 }
+
+func (v Vpc) nukePeeringConnections() error {
+	peeringConnections, err := v.svc.DescribeVpcPeeringConnections(
```

Comment:
This function will only return the first page of peering connections. In the (admittedly unlikely) event that there is more than a single page, errors will occur. Can you swap this out with the paginated version of this function? [aws docs](https://docs.aws.amazon.com/sdk-for-go/api/service/ec2/#EC2.DescribeVpcPeeringConnectionsPages)

Reply:
done